test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -129,6 +129,11 @@ public EndToEndTests(ServerFixture serverFixture, ITestOutputHelper output) : ba
 
                     await closeTcs.Task.OrTimeout();
                 }
+                catch (Exception ex)
```

Comment:
Let's leave it for now and we will see if it is bad. This might be helpful when investigating flaky tests in the CI - losing the original exception will make it hard to figure out what happened.